siteimprove/admin/js/siteimprove-remove-adminbar.js

```diff
@@ -0,0 +1,7 @@
+window.onload = function() {
```

Comment:
maybe change this to `window.addEventListener("load",function(){ `so that we don't take over the whole window.onload, and instead allow for other things to happen also as part of this event.